galaxy_ng/tests/integration/api/test_rbac_roles.py

```diff
@@ -297,6 +297,7 @@
 
 
 @pytest.mark.role_rbac
+@pytest.mark.standalone_only
```

Comment:
for `insights` mode we want to make sure this test is run for roles applicable to `insights` mode

Reply:
since these tests create a user via the api, and that is not available in `insights` mode, this is :+1: